
The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- galan-0.2.14/src/event.c.orig	2004-05-13 20:36:28.045674776 +0200
+++ galan-0.2.14/src/event.c	2004-05-13 20:37:54.848478744 +0200
@@ -55,6 +55,7 @@
 		memcpy( dst->d.array.numbers, src->d.array.numbers, src->d.array.len * sizeof(gdouble) );
 		break;
 	    default:
+		;
 	}
     }
 	
@@ -73,6 +74,7 @@
 		    free( e->d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( e );
     }
@@ -91,6 +93,7 @@
 		    free( evq->e.d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( evq );
     }
